src/main/java/me/minidigger/hangar/service/project/ProjectFactory.java

```diff
@@ -64,4 +72,16 @@ public ProjectsTable createProject(UsersTable ownerUser, String name, String plu
 
         return projectsTable;
     }
+
+    public enum InvalidProject {
```

Comment:
not sure thats a nice name, but oh well

Reply:
can refactor easily, maybe InvalidProjectReason?